alerts/alerts.libsonnet

```diff
@@ -24,6 +24,43 @@
           },
         ],
       },
+      {
+        name: 'mintel-disks',
```

Comment:
what about the Inodes rule for standard , non volumes, disks and the usage for kubelet volumes ?